numpy/lib/tests/test_format.py

```diff
@@ -838,7 +838,11 @@ def test_large_file_support():
 
 @dec.slow
 def test_large_archive():
-    a = np.empty((2 ** 30, 2), dtype=np.uint8)
+    try:
+        a = np.empty((2**26, 2), dtype=np.uint8)
```

Comment:
Thinking a bit more, this really only needs to be run on 64 bit windows systems.